src/Analyzer.Utilities.UnitTests/JsonPathResolverTests.cs

```diff
@@ -104,19 +101,15 @@ public void Resolve_JsonContainsPath_ReturnsResolverWithCorrectJtokenAndPath(str
                     },
                 }");
 
-            var resolver = new JsonPathResolver(jtoken, jtoken.Path);
+            var resolver = new JsonPathResolver(jobject, jobject.Path);
 
-            // Do twice to verify internal cache correctness
```

Comment:
should this checking still be here ("// Do twice to verify internal cache correctness")?

Reply:
and likewise is other spots in this file?

Reply:
Sounds like a great improvement, thank you for explaining!